resource_customizations/argoproj.io/Rollout/actions/action_test.yaml

```diff
@@ -11,6 +11,8 @@ discoveryTests:
       disabled: true
     - name: promote-full
       disabled: true
+    - name: pause
+      disabled: false
```

Comment:
Should this be true, since it seems to be paused based on the name of the input path?